src/main/java/dev/ebrydeu/spring_boot_library/domain/entities/User.java

```diff
@@ -37,9 +37,9 @@ public class User extends Auditable {
     @Column(nullable = false, unique = true)
     private String email;
 
-    @ManyToOne(cascade = CascadeType.ALL)
-    @JoinColumn(name = "role_id")
-    private Role role;
+    @Enumerated(EnumType.STRING)
+    @Column(name = "role", columnDefinition= "VARCHAR(20) Default 'STANDARD")
+    private Enum role;
```

Comment:
In column definitions, either always use uppercase or lowercase letters; do not mix them.
```suggestion
    @Enumerated(EnumType.STRING)
    @Column(name = "role", columnDefinition= "VARCHAR(20) DEFAULT 'USER'")
    private Role role;
```